certora/steward/specs/rules.spec

```diff
@@ -289,19 +289,19 @@ rule updateGhoBorrowCap__correctness() {
     assert BORROW_CAP==newBorrowCap;
 
     uint256 borrow_cap_after = BORROW_CAP;
-    assert borrow_cap_before <= borrow_cap_after && to_mathint(borrow_cap_after) <= 2*borrow_cap_before;
+    assert 0 <= to_mathint(borrow_cap_after) && to_mathint(borrow_cap_after) <= 2*borrow_cap_before;
```

Comment:
This is a correct conversion.
Note that the first clause is trivially correct since `exposure_cap_after` is a uint128 and the solidity will make sure to revert if the update function cause an underflow.
That's still fine to leave in.

Reply:
fixed in [cdb664e](https://github.com/aave/gho-core/pull/406/commits/cdb664ef243baad64786b6d89fb0a1dc21e850a5)